translate/convert/web2py2po.py

```diff
@@ -36,9 +36,9 @@ def __init__(self, pofile=None):
 
     def convertunit(self, source_str, target_str):
         pounit = po.pounit(encoding="UTF-8")
-        pounit.setsource(source_str)
+        pounit.source = source_str
         if target_str:
-            pounit.settarget(target_str)
+            pounit.target = target_str
```

Comment:
Please squash this commit into first commit in PR since it fixes an usage for a deprecated API that you are introducing there. This way you will be contributing clean code.